src/item_factory.cpp

```diff
@@ -846,7 +846,7 @@ void Item_factory::finalize_post( itype &obj )
 
                 // need to account for varsize stuff here and double encumbrance if so
                 if( obj.has_flag( flag_VARSIZE ) ) {
-                    data.encumber *= 2;
+                    data.encumber = std::min( data.encumber * 2, data.encumber + 10 );;
```

Comment:
Will this affect all clothing items that have variable sizes, i.e. make poorly fitted t-shirts have 10 encumbrance? That'd be a pretty big change, so if that's the case, I'd probably point that out in the PR description and maybe the changelog.

Reply:
Nope! This is actually just code to counteract variable size. Since this helmet stuff for Aerin calculates the right value I have to then increase the value so that later when it fits and is reduced, it ends up at the value calculated here.

Reply:
Gotcha, cool beans.